global.json

```diff
@@ -0,0 +1,6 @@
+{
+    "sdk": {
+      "version": "8.0.101",

```

Comment:
```suggestion
      "version": "8.0.400",

```

Reply:
That as on purpose not to "force" users to have a newer version. And the `rollForward` will still take the newer version if you have it.

This way also we don't have to update the file with every single new version.
We could pick another more recent version too if we have a reason, like high security patch that we require users to use (why would "we" force them though).